opencl/CMakeLists.txt

```diff
@@ -78,4 +78,24 @@ endif()
 FetchContent_MakeAvailable(ocl-icd)
 add_library(OpenCL-ICD ALIAS OpenCL)
 
+# OpenCL RT build requires headers from ICD loader, propagating location
+set(OpenCL_LOADER_DIR
+  ${ocl-icd_SOURCE_DIR} CACHE PATH "Path to OpenCL Loader")
+
+# TODO: Remove suppressing Werror.
+if(MSVC)
+  target_compile_options(OpenCL PRIVATE "-WX-")
+else()
+  target_compile_options(OpenCL PRIVATE "-Wno-error")
+endif()
```

Comment:
Why is this code here? As of today, we are able to build with `-Werror` with no problem.

Reply:
All this PR is the code used in xmain.  I just remove the INTEL* macro.  I will verify it and remove it.

Reply:
It's removed.